src/main.cpp

```diff
@@ -9,28 +9,23 @@
 
 #include "claw.hpp"
 #include "claw_state.hpp"
-#include "uart_connection.hpp"
-
-inline void debugUarmRx(UARTConnection &conn);
+#include "uart_lib.hpp"
 
 namespace target = hwlib::target;
 
-long startMsReceive = 0, startMsSend = 0;
-
 int main() {
```

Comment:
Main could use some cleanup. It won't be pushed like this to master anyway.

Reply:
Because it's quite tricky to determine the claw's state, it's important that the demo works this way. Unfortunately this requires many lines of code, all unnecessary lines are already removed.